orchagent/saihelper.cpp

```diff
@@ -27,6 +28,8 @@ using namespace swss;
 #define STR(s) _STR(s)
 
 #define CONTEXT_CFG_FILE "/usr/share/sonic/hwsku/context_config.json"
+#define SAI_REDIS_GET_RESPONSE_TIMEOUT_MS_LONG (480*1000)
+#define SAI_REDIS_GET_RESPONSE_TIMEOUT_MS_DEFAULT (60*1000)
```

Comment:
Is it possible to includes sai redies's header so that we don't need to redefine the same value for the second time?
It will be great if we can read the configuration from some configuration file instead of headcoding it.

Reply:
@stephenxs Done.